### PR TITLE
various changes

### PR DESCRIPTION
**Pull request by DavidBJaffe**
- Add debugging lines to attempt to trap a sporadic failure that is observed in the CI test.
- Refactor computation of the diff row.  This fixed a couple bugs, including cases where the
   diff row content was missing, and also the handling of it when `COLOR=property` was specified.
- Add new option DIFF_STYLE.

### CODE REVIEW

**Comment by rjrico10x** [comment]
jenkins retest this please

**Comment by rjrico10x** [comment]
checking a change I made in the jenkins script

**Comment by rjrico10x** [comment]
jenkins retest this please

**Comment by rjrico10x** [comment]
trying another change to the jenkins script to make it a bit less verbose

File: enclone_help/src/help4.rs [hunk 1]
```diff
@@ -77,13 +77,13 @@ pub fn help4(args: &Vec<String>, mut h: &mut HelpDesk) {
...
             "NUMI",
-            "Filter out B cells based on low BCR UMI counts.  The heuristics for this",
+            "Filter out B cells based on low BCR UMI counts.  The heuristics",
             65,
         );
 
         h.docpr(
             "",
-            "are described on the enclone site at \\green{bit.ly/enclone}.",
+            "for this are described on the enclone site at \\green{bit.ly/enclone}.",
```
Comment (wyattmcdonnell) [inline enclone_help/src/help4.rs:86]: Can we put in the link to the heuristics HTML page here instead so people don't have to look for it?

**Review by wyattmcdonnell — APPROVED** [review]
Nice refactor. Good to go!

**Pull request merged**